bqn-mode.el

```diff
@@ -218,69 +218,63 @@
 
 ;;;; core functionality
 
+(defface bqn-default
```

Comment:
Given that you are cogitating to remove the redundant `-face` suffix, my suggestion is a bit convoluted and certainly a trace of Obsessive-Compulsive Disturbance. Please tell me if it makes sense.

We have the following options:

- [ ] remove all redundant `-face` suffixes now (and implement `bqn-default` later in another commit or PR)
- [x] use `bqn-default-face` now (and remove remove all redundant `-face` suffixes later in another PR or commit)

This is because it will be easier to explain in the commit messages why you are doing this or that.  

Ideally each commit should change one thing. It will be easier to follow the history later.

Reply:
I agree on the single-change thing, that's precisely what I am trying to do.  But using -face is just not appropriate (see the comment below and the docs), so using the suffix here is like intentionally adding a bug for "consistency".